### PR TITLE
Devhub 1025

### PR DESCRIPTION
**Pull request by harikakotipalli**
provides dynamic folder structure for L1's and hierarchy level for L2's while using same file [...slug].tsx

check Preview link 
https://devcenter-au4fduoxb-harikakotipalli.vercel.app/product/atlas
https://devcenter-au4fduoxb-harikakotipalli.vercel.app/product/atlas/change-streams
https://devcenter-au4fduoxb-harikakotipalli.vercel.app/language/c

### CODE REVIEW

File: src/pages/[l1_l2]/[...slug].tsx [hunk 1]
```diff
@@ -173,20 +174,29 @@ const Topic: NextPage<TopicProps> = ({
 export default Topic;
 
 interface IParams extends ParsedUrlQuery {
-    slug: string;
+    l1_l2: string;
+    slug: string[];
```
Comment (harikakotipalli) [inline src/pages/[l1_l2]/[...slug].tsx:178]: main change slug is array

l1_l2 = product , slug = ['a'] will result in route /product/a
l1_l2 = product , slug = ['a', 'b'] will result in route /product/a/b

File: src/data/l2-products.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Taxonomy } from '../interfaces/taxonomy';
+
+export const l2Products: Taxonomy[] = [
+    {
+        name: 'Change Streams',
+        slug: 'atlas/change-streams',
```
Comment (harikakotipalli) [inline src/data/l2-products.ts:6]: backend should construct slugs for l2 in this way

**Review by mcolella14 — APPROVED** [review]
Looks good to me!

**Review by gfunk1230 — APPROVED** [review]
LGTM!

**Pull request merged**